backend/src/apps/owasp/api/internal/mutations/certificate.py

```
@@ -0,0 +1,131 @@
+            not user.github_user.is_project_leader and not user.github_user.chapters.exists()
+        ):
+            msg = "You must be a project leader or chapter leader to issue certificates."
+            logger.warning(
+                "Permission denied for user '%s' to issue a certificate.",
+                user.username,
+            )
+            raise PermissionDenied(msg)
+
+        logins = []
+        if input_data.recipient_logins:
+            logins = [l.strip() for l in input_data.recipient_logins if l and l.strip()]
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)